code/src/componenets/ThoughtForm.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+const ThoughtForm = ({onFormSubmit, newThought, setNewThought, setCount, count}) => {
+
+const checkKey = (event) => {
+    if (event.keyCode === 13 && !event.shiftKey) {
+        onFormSubmit(event)
+    }
```

Comment:
I tried to get this to work, but i didn't manage. Might try what you did, thank you!